services/graph/pkg/config/config.go

```diff
@@ -94,6 +94,8 @@ type LDAPEducationConfig struct {
 	SchoolNameAttribute   string `yaml:"school_name_attribute" env:"GRAPH_LDAP_SCHOOL_NAME_ATTRIBUTE" desc:"LDAP Attribute to use for the name of a school."`
 	SchoolNumberAttribute string `yaml:"school_number_attribute" env:"GRAPH_LDAP_SCHOOL_NUMBER_ATTRIBUTE" desc:"LDAP Attribute to use for the number of a school."`
 	SchoolIDAttribute     string `yaml:"school_id_attribute" env:"GRAPH_LDAP_SCHOOL_ID_ATTRIBUTE" desc:"LDAP Attribute to use as the unique id for schools. This should be a stable globally unique ID like a UUID."`
+
+	SchoolTerminationGraceDays int `yaml:"school_termination_min_grace_days" env:"GRAPH_LDAP_SCHOOL_TERMINATION_MIN_GRACE_DAYS" desc:"When setting a termination date for a school, require the date to be at least this number of days in the future."`
```

Comment:
This is getting strange.
* First we have days which point to the future. (ok, my suggestion would need a second round too)
* Now this reads as a date, format to be documented, where it is clear that a date can be falsly set to the past which is not possible with days.
* We need to add text telling to disable it by setting it to emty.

Reply:
> * First we have days which point to the future. (ok, my suggestion would need a second round too)

 The `GRAPH_LDAP_SCHOOL_TERMINATION_MIN_GRACE_DAYS` setting is always a number of days. (That value can't be a date)

>     * Now this reads as a date, format to be documented, where it is clear that a date can be falsly set to the past which is not possible with days.

* The termination date of a school is set via a request on the provisioning API, see the examples in the initial description (or the examples on the swagger page for the libregraph API). The format is basically what the OpenAPI `date-time` string format supports, as the defined in the spec.

>  * We need to add text telling to disable it by setting it to empty.

Just not setting this, disabled it. (The default is 0 days, but I can add that to the test.